spec/contact_sync_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe ContactSync do
+  it 'does stuff' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.